src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -1,25 +1,62 @@
 
-Developers can use this SDK to interact with the [Configuration Store](https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-dotnet-core-app#create-an-app-configuration-store) where the configuration settings are stored.
+Use the client library for App Configuration to:
+- Create a Configuration Setting
+- Retrieve a Configuration Setting
+- Update an existing Configuration Setting
+- Delete a Configuration Setting
 
-Actions that can be executed are:
+[Source code]() | [Package (NuGet)][package] | [API reference documentation]() | [Product documentation][azconfig_docs]
 
-- Perform basic reads, writes, updates, and deletes of an application configuration settings.
-- Get the history of a configuration setting.
-- Watch for changes in a specific configuration setting.
+## Getting started
 
-# Installing
+### Install the package
 
-A NuGet package called Azure.ApplicationModel.Configuration will be avaliable soon.
+Install the Azure App Configuration client library for .NET with [NuGet][nuget]:
 
-# Configuration Setting
+```Powershell
+Install-Package Azure.ApplicationModel.Configuration -Version 1.0.0-preview.1
+```
+
+**Prerequisites**: You must have an [Azure subscription][azure_sub], and a [Configuration Store][configuration_store] to use this package.
+
+To create a Configuration Store, you can use the Azure Portal or [Azure CLI][azure_cli] with the following command:
+
+```Powershell
+az appconfig create --name <config-store-name> --resource-group <resource-group-name> --location eastus
+```
+
+### Authenticate the client
+
+In order to interact with the App Configuration service, you'll need to create an instance of the [Configuration Client][configuration_client_class] class. To make this possible, you'll need the connection string of the Configuration Store.
+
+#### Get credentials
+Use the [Azure CLI][azure_cli] snippet below to get the connection string from the Configuration Store.
+```Powershell
+az appconfig credential list --name <config-store-name>
+```
+
+Alternatively, get the connection string from the Azure Portal.
+
+#### Create client
+
+Once you have the value of the connection string, you can create the ConfigurationClient:
+
+```c#
+string connectionString = <connection_string>;
+var client = new ConfigurationClient(connectionString);
+```
+
+## Key concepts
+
+### Configuration Setting
 Is the fundamental resource within a Configuration Store. In its simplest form it is a key and a value. However, there are additional properties such as the modifiable content type and tags fields that allow the value to be interpreted or associated in different ways.
```

Comment:
Might be redundant with the header of the section being Configuration Setting but it reads cleaner starting with "Configuration Setting is the fundamental..."